web-server-test/tests/web-server/e2e/read-write/tests.rkt

```diff
@@ -27,11 +27,15 @@
     "sending data too slowly"
     broken-pipe?
     (lambda _
-      (define-values (in out)
-        (tcp-connect "127.0.0.1" port))
+      ;; On Racket CS 7.4 the default plumber writes an error to
+      ;; standard out when it tries to close the socket. Creating a
+      ;; custom plumber seems to fix that problem.
+      (parameterize ([current-plumber (make-plumber)])
```

Comment:
Is this error still there on RacketCS HEAD? Is it a bug?

Reply:
I think it might be. IIRC tests on the latest CS snapshot did not fail because of this, but I'm going to have to double check tomorrow.